lib/icinga/checkable-flapping.cpp

```diff
@@ -64,10 +64,16 @@ void Checkable::UpdateFlappingStatus(bool stateChange)
 	SetFlappingBuffer(stateChangeBuf.GetValue());
 	SetFlappingIndex(oldestIndex);
 	SetFlappingCurrent(flappingValue);
-	SetFlapping(flapping, true);
 
-	if (flapping != GetFlapping())
-		SetFlappingLastChange(Utility::GetTime());
+	if (flapping != GetFlapping()) {
+		SetFlapping(flapping, true);
+
+		double ee = GetLastCheckResult()->GetExecutionEnd();
```

Comment:
Why the change from `now` to `execution_end`?

Reply:
Each node calcs flapping – and "now" – by itself, but we need a consistent ID based on this across nodes.